deployment/localsettings.template.py

```diff
@@ -15,3 +17,18 @@
 
 # Make apache work when DEBUG == False
 ALLOWED_HOSTS = ["localhost", "127.0.0.1"]
+
+### Evaluation progress rewards
+GLOBAL_EVALUATION_PROGRESS_REWARDS: list[tuple[float, str]] = [
+    (0, "0€"),
+    (0.25, "1.000€"),
+    (0.6, "3.000€"),
+    (0.7, "7.000€"),
+    (0.9, "10.000€"),
```

Comment:
I thought about using `Decimal` instead of float here. Since we always round/ceil, any decimal part inconsistency doesn't really matter, so I decided against it. I think the only thing we could run into is numers so big that they can not be represented by floats anymore, but I doubt we'll ever get close to 10^15 votes/participations. Am I missing something?

I'm still open to using `Decimal` or `Fraction`, just thought it would be less canonical with no advantage.

Reply:
I am only worrying about cases where we jump from one integer to the other, so for ceiling, cases where the float multiplication ends up at 500 vs 500.00001. I am not super sure whether any grave problems could actually occur, I think I would slightly prefer fractions just for the peace of mind though

Reply:
Fair. Problem example, for a possible test case:
```python3
>>> print(0.07 * 100, math.ceil(0.07 * 100))
7.000000000000001 8
```

Reply:
tested that this cannot happen with `max_reward_votes` and `next_reward_remaining_votes` are rounded correctly. The template filter is okay, because it uses round and `round(float("0.XX")*100) == XX` for all XX < 100.

Reply:
with the current design, this is not easily tested anymore; the only position where `max_reward_votes` is used is indirectly for `width-percent-XX`, where we would have a rounding error (`voter_count=4`, `participant_count=100`, `Fraction("0.07")`)